docs/userguide.rst

```diff
@@ -10,9 +10,11 @@ REANA cluster uses `Kubernetes <https://kubernetes.io/>`_ container
 orchestration system. The best way to try it out locally on your laptop is to
 install:
 
-- ``kubectl`` 1.16.3 or higher (see `kubectl installation guide <https://kubernetes.io/docs/tasks/tools/install-kubectl/>`_)
-- ``minikube`` 1.5.2 or higher (see `minikube installation guide <https://kubernetes.io/docs/tasks/tools/install-minikube/>`_)
-- ``helm`` 3.0.0 or higher (see `helm installation guide <https://helm.sh/docs/using_helm/#installing-helm>`_)
+- ``kubectl`` e.g. 1.16.3 (see `kubectl installation guide <https://kubernetes.io/docs/tasks/tools/install-kubectl/>`_)
+- ``minikube`` e.g. 1.5.2 (see `minikube installation guide <https://kubernetes.io/docs/tasks/tools/install-minikube/>`_)
+- ``VirtualBox`` e.g. 6.1.0 (see `VirtualBox installation guide <https://www.virtualbox.org/manual/ch02.html>`_)
```

Comment:
Sort them alphabetically and perhaps all names in lowercase (indicating package names)?